earth_enterprise/src/fusion/portableglobe/servers/local_server.py

```diff
@@ -388,21 +388,65 @@ def LocalJsonSearchHandler(self, handler):
         search_term = handler.request.arguments[key][0].lower()
         handler.write(tornado.web.globe_.search_db_.JsonSearch(search_term, cb))
 
+  def ParseGlobeReqName(self, req):
```

Comment:
Since we updated this file, let's update copyright to include Open GEE Contributors with 2019 year:
Copyright 2017 Google Inc, 2019 Open GEE Contributors